### PR TITLE
Update messages_no.js

### PR DESCRIPTION
**Pull request by Nettsentrisk**
Cleaning up and simplifying Norwegian translations, removing Swedish non-standard translations for validators not included in project source code.

### CODE REVIEW

**Comment by staabm** [comment]
looks good to me, thx.

@Arkni any comments?

File: src/localization/messages_no.js [hunk 1]
```diff
@@ -3,23 +3,21 @@
...
 	digits: "Skriv kun tall.",
 	equalTo: "Skriv samme verdi igjen.",
+	maxlength: $.validator.format( "Maksimalt {0} tegn." ),
+	minlength: $.validator.format( "Minimum {0} tegn." ),
+	rangelength: $.validator.format( "Angi minimum {0} og maksimum {1} tegn." ),
 	range: $.validator.format( "Angi en verdi mellom {0} og {1}." ),
 	max: $.validator.format( "Angi en verdi som er mindre eller lik {0}." ),
 	min: $.validator.format( "Angi en verdi som er st&oslash;rre eller lik {0}." ),
-	step: $.validator.format( "Angi en verdi ganger {0}." ),
+	step: $.validator.format( "Angi en verdi ganger {0}."),
```
Comment (Arkni) [inline src/localization/messages_no.js]: Please revert this change. The space character should remain as it follows our coding style.

**Comment by Nettsentrisk** [comment]
OK, fixed.

**Comment by staabm** [comment]
thank you guys.
